src/util/log.h

```diff
@@ -31,8 +31,8 @@ enum class LogLevel {
 
 #define STR(x) #x
 
-#define ASSERT(pass, msg, ...) _smek_assert(__FILE__, __LINE__, __func__, pass, STR(pass), msg, ##__VA_ARGS__)
-#define CHECK(pass, msg, ...)  _smek_check(__FILE__, __LINE__, __func__, pass, STR(pass), msg, ##__VA_ARGS__)
+#define ASSERT(pass, ...) _smek_assert(__FILE__, __LINE__, __func__, pass, STR(pass), __VA_ARGS__)
+#define CHECK(pass, ...)  _smek_check(__FILE__, __LINE__, __func__, pass, STR(pass), __VA_ARGS__)
```

Comment:
While it doesn't look like it (which might be a problem, admittedly) this has the same requirement. If you don't pass any parameters it won't compile since there's a trailing comma in the function call. `##__VA_ARGS__` which is needed otherwise is a GCC-extension that isn't actually needed.